src/interface/interface.json

```diff
@@ -186,6 +186,85 @@
                                 "profileSettingsLabel": "{String} profileSettingsLabel",
                                 "signOutLabel": "{String} signOutLabel"
                             }
+                        },
+                        "ProjectAccountSwitcher": {
+                            "methods": {
+                                "addAccount": {
+                                    "desc": "Pass in an instance of a ProjectAccountSwitcher Account",
+                                    "params": [
+                                        "{Account} [ReferenceAccount]",
+                                        "{Account} [ReferenceAccount] - (Optional) reference node for insertBefore"
+                                    ]
+                                },
+                                "addProject": {
+                                    "desc": "Pass in an instance of a ProjectAccountSwitcher Project",
+                                    "params": [
+                                        "{Project} [ReferenceProject]",
+                                        "{Project} [ReferenceProject] - (Optional) reference node for insertBefore"
+                                    ]
+                                },
+                                "open": {
+                                    "desc": "opens the profile"
+                                },
+                                "close": {
+                                    "desc": "closes the profile"
+                                },
+                                "setActive": {
+                                    "desc": "Pass an instance of either Account or Project",
+                                    "params": [
+                                        "{Account or Project} - The account or project currently chosen"
+                                    ]
+                                },
```

Comment:
The only thing I wonder about here is that projects and accounts have different shaped images. Do I let consumers set an image of any aspect ratio? If not, I also need a method to set if the active thing is a project or an account.